packages/@uppy/webcam/src/Webcam.tsx

```diff
@@ -501,26 +609,32 @@ export default class Webcam extends UIPlugin {
       return {
         source: this.id,
         name,
-        data: new Blob([blob], { type: mimeType }),
+        data: new Blob([blob!], { type: mimeType }),
```

Comment:
Non-null assertions should really be the exception rather than the rule imo. I see them several times in this file and it can lead to unsafe code. Often it's easy enough to add an if statement to check if the variable is actually there, could that be done here?